### DISCUSSION TITLE
docker - invalid database

### DISCUSSION THREAD

**Discussion by EVOTk** [filebrowser/filebrowser#1192] [Q&A]
**Description**
I want to install filebrowser in docker on RancherOS

**Expected behaviour**
Docker start normally

**What is happening instead?**
`2020/12/19 16:23:48 invalid database`

**Additional context**
I'm create filebrowser.db with vi
Create docker : 
`sudo docker run \
    -v /home:/srv \
    -v /home/docker/filebrowser/db/filebrowser.db:/database.db \
    -p 8080:80 \
    filebrowser/filebrowser`

The logs : 

```
Unable to find image 'filebrowser/filebrowser:latest' locally
latest: Pulling from filebrowser/filebrowser
c17ea15b6e0b: Pull complete
6703361741bf: Pull complete
d2df7c4ef3bd: Pull complete
67bd4ed5b955: Pull complete
Digest: sha256:590852a42a8fde7fe801353958d8d97107efa800cad3466e1e44af0f92d8e08c
Status: Downloaded newer image for filebrowser/filebrowser:latest
2020/12/19 16:23:48 invalid database
```

Thx !

**Comment by o1egl** [comment]
Database file must be created before starting docker container. You can do it with `touch /home/docker/filebrowser/db/filebrowser.db`
